components/x-teaser/storybook/article.js

```diff
@@ -36,6 +36,7 @@ exports.knobs = [
 	'showImage',
 	'image',
 	'imageSize',
+  'imageHighestQuality',
```

Comment:
it wasn't just that one line that had spaces, all the indentation in the PR is! 😬

Reply:
😱 😱